oaknational/Oak-Web-Application
# Feat/pupil 1103/create feature flags

## Description

Creating feature flags to match those in production posthog

## How to test

1. Go to https://deploy-preview-3037--oak-web-application.netlify.thenational.academy/teachers/lessons/transverse-waves
2. Launch the posthog tool bar
3. check the following feature flags between test and control and no flag :
- share-advocate-lesson (on a lesson page)
- share-advocate-download (on a download confirmation page)
- share-advocate-unit (on a lesson listing page)
------------------------
- no flag: should not display the button and not update the url and not fire any teacher share events
- control: should not display the button but should update the url and should fire teacher share initiated
- test: should display the button, update the url, fire teacher share initiated and teacher share activated when the button is pressed.
